src/black_primer/lib.py

```diff
@@ -39,7 +39,7 @@ class Results(NamedTuple):
 
 async def _gen_check_output(
     cmd: Sequence[str],
-    timeout: float = 300,
+    timeout: float = 600,
```

Comment:
Maybe we could also move this to be per project ... So we don't wait for ever for small projects ...